src/main/java/com/GeorgesServer/app/com/GeorgesServer/handler/IHandler.java

```diff
@@ -3,6 +3,12 @@
 import com.GeorgesServer.app.com.GeorgesServer.request.ClientRequest;
 import com.GeorgesServer.app.com.GeorgesServer.response.ServerResponse;
 
+import java.util.HashMap;
+
 public interface IHandler {
```

Comment:
Adding getBody(), getHeaders(), getStatus(), format() to the IHandler interface allows each concrete implementation  to have it's own custom setters that will provide the values for each of these getters